client/components/Navbar.js

```diff
@@ -25,23 +36,23 @@ const Navbar = ({handleClick, isLoggedIn}) => (
     </nav>
     <hr />
   </div>
-)
+);
 
 /**
  * CONTAINER
  */
-const mapState = state => {
+const mapState = (state) => {
   return {
-    isLoggedIn: !!state.auth.id
-  }
-}
+    isLoggedIn: !!state.auth.id,
```

Comment:
So that was actually in the starter code (we didn't put that in there). We looked it up and double exclamations convert truthy or falsy data points to true or false (i.e. if it was 7 it would convert it to true, if it was 0 it would convert it to false).